docs/modules/reference/pages/model.adoc

```diff
@@ -1,17 +1,17 @@
 = Model
-:imagesdir: ../assets/images 
+:imagesdir: ../assets/images
 
 == Types
 
-The following types are supported using the built in Inventory Scripts:
+The following types are supported using the built-in inventory scripts:
 
 === Node (node)
```

Comment:
Maybe we could use the `Node:: <definition>` syntax for this section; it might look cleaner than having many short sections.